prow/scripts/install_deps_debian.sh

```diff
@@ -0,0 +1,55 @@
+#!/bin/bash
+
+###
+# Following script installs necessary tooling for Debian to deploy Kyma on Minikube.
+#
+# REQUIREMENTS:
+# 64-bit version of one of these Debian versions:
+# 
+# - Stretch 9 (stable)
+# - Jessie 8 (LTS)
+# 
+###
+
+set -o errexit
+
+MINIKUBE_VERSION=v0.28.2
+KUBECTL_CLI_VERSION=v1.10.0
+HELM_VERSION=v2.8.2
+DOCKER_VERSION=18.06.1~ce~3-0~debian
+
+# install docker
+sudo apt-get update
+sudo apt-get install -y \
+     apt-transport-https \
+     ca-certificates \
+     curl \
+     gnupg2 \
+     socat \
+     software-properties-common
+
+curl -fsSL https://download.docker.com/linux/debian/gpg | sudo apt-key add -
```

Comment:
Instead of installing it every time, maybe we should prepare a custom VM image?

Reply:
We already have an issue to investigate this [here](https://github.com/kyma-project/test-infra/issues/47). So, this shouldn't be a blocker for this PR.